### DISCUSSION TITLE
Share to instant message tool (example slack ) feature

### DISCUSSION THREAD

**Discussion by jayhyuk** [grafana/grafana#63993] [General]
Hi , grafana team , i have implement some feature in my own grafana repo. which
help us to easy to share panel to other im tool 

not sure if you think this feature should be exist in grafana one ?
![image](https://user-images.githubusercontent.com/12393886/222334375-2dc62119-d119-415e-8a18-9807ad70f7e1.png)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82789. Feel free to continue the discussion around this there. Thank you!
